### ISSUE TITLE
Add flexible config for editor buttons

### ISSUE THREAD

**Issue by buzzedword** [jhollingworth/bootstrap-wysihtml5#27]
Think we should add some plain english config for the toolbar buttons. You already have a nice editor-- no need to convolute it by making folks edit the template to add more options. Maybe something like an array of options to include? Like--

``` javascript
   $('#textarea').wysihtml5("simple|advanced", 
   ['link', 'outline', 'heading'], 
   { headingName: 'className' });
```

Just a thought. What say you, @jhollingworth?

**Comment by jhollingworth** [comment]
Yeah - makes complete sense. I'd like to expose everything that wysihtml5 does via the options object. Just gotta find the time :)
